### PR TITLE
SC-1840: Update dependencies and modify msg field text

### PR DESCRIPTION
**Pull request by ehernand**

### CODE REVIEW

File: mvnw [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+fi
+
+if [ ! -x "$JAVACMD" ] ; then
+  echo "Error: JAVA_HOME is not defined correctly." >&2
+  echo "  We cannot execute $JAVACMD" >&2
+  exit 1
+fi
+
+if [ -z "$JAVA_HOME" ] ; then
+  echo "Warning: JAVA_HOME environment variable is not set."
```
Comment (sjayrams) [inline mvnw:159]: exit 1?

File: slf4j-json-logger/src/main/java/com/savoirtech/logging/slf4j/json/logger/StandardJsonLogger.java [hunk 2]
```diff
@@ -102,21 +102,21 @@ public void setIncludeClassName(boolean includeClassName) {
...
     catch (Exception e) {
-      jsonObject.add("message", gson.toJsonTree(formatException(e)));
+      jsonObject.add("msg", gson.toJsonTree(formatException(e)));
     }
     return this;
   }
 
   @Override
   public JsonLogger message(Supplier<String> message) {
     try {
```
Comment (sjayrams) [inline slf4j-json-logger/src/main/java/com/savoirtech/logging/slf4j/json/logger/StandardJsonLogger.java:115]: return this.message(message.get())?

File: mvnw [hunk 3]
```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```
Comment (sjayrams) [inline mvnw:1]: add to readme on how to use this?

**Review by sjayrams — APPROVED** [review]

**Review by shershkop — CHANGES_REQUESTED** [review]
we aren't going to import/fork outside libs. please find another way

**Comment by ehernand** [comment]
Closing PR since it's not needed anymore

**Pull request closed without merging**